### PR TITLE
HACKATHON: Add calendarhero api

### PR DESCRIPTION
**Pull request by richard-rance**
Add documentation for the calendar hero APIs that are routed through the api-gateway as a proxy in [#179](https://github.com/vendasta/api-gateway/pull/179)

You can preview the docs on this branch [here](https://developers.vendasta.com/platform/branches/add-calendarhero-api/ed58ca998910a-calendar-hero-api)

Note: I don't want to merge this until we add actual documentation to each of the fields and APIs. We should also decide what to do about Vendasta users that should be able to access multiple CalendarHero instances and partner service accounts.

https://github.com/orgs/vendasta/teams/external-apis @jaycroll @Alkasai

### CODE REVIEW

File: openapi/calendar/calendar.yaml [hunk 1]
```diff
@@ -0,0 +1,1535 @@
...
+              - q
+              - r
+              - s
+              - t
+              - u
+              - v
+              - w
+              - x
+              - 'y'
+              - z
```
Comment (Alkasai) [inline openapi/calendar/calendar.yaml:66]: What's this?

Comment (Alkasai) [reply openapi/calendar/calendar.yaml:66]: Is this file just an example of how to add CH documentation?

Comment (richard-rance) [reply openapi/calendar/calendar.yaml:66]: I imported the existing API docs from https://api.calendarhero.com/documentation and then changed the server and auth details. 

To maintain this file going forward we have two options:
1. Improve the existing CH API docs and set up a build process to automatically make the same changes that I did while copying updates to this repo.
2. Use this repo as the source of truth and follow [this guide](https://vendasta.jira.com/wiki/spaces/API/pages/1670709369/Updating+Documentation#Stoplight-Content) to edit the docs using Stoplight Studio.

If we decide to change from a raw proxy to actually rewriting the request and response bodies (so we can filter out some deprecated fields) then option 2 is the way to go.

**Comment by jredl-va** [comment]
closing old prs

**Pull request closed without merging**